bootstrap_cfn/fab_tasks.py

```diff
@@ -420,3 +420,25 @@ def get_cloudformation_tags():
     return {
         "Env": env.environment,
     }
+
+
+@task
+def display_elb_dns_entries():
+    """
+    Prints out the ELB name(s) and the corresponding DNS name(s) for every ELB
+    in the environment provided.
+    """
+    stack_name = get_stack_name()
+    if not stack_name:
+        abort("Could not retrieve stack_name")
+    elb = get_connection(ELB)
+    if not elb:
+        abort("Could not retrieve elb object")
+    elb_dns_list = elb.list_domain_names(stack_name)
+    import json
```

Comment:
I thought formatted json would be humanly readable and it could be used down the chain